style.css

```diff
@@ -165,6 +165,13 @@ p {
   justify-content: flex-end;
```

Comment:
PR should have a summary of features implemented.
Please do not merge the branch with the main branch before approval from a code reviewer.

Reply:
I already deployed my project a long time ago. And when I tried to copy the link of my readme file to submit the request it doesn't accept it.